.dockerignore

```diff
@@ -0,0 +1 @@
+hack/libgit2/
```

Comment:
I think this should be `libgit2-static`, or at least this is the dir created on macOS.